src/test_l1_tvSettings.c

```diff
@@ -427,13 +421,12 @@ void test_l1_tvSettings_negative_RegisterVideoFormatChangeCB (void)
  * | Variation / Step | Description | Test Data | Expected Result | Notes |
  * | :-------: | ------------- | --------- | --------------- | ----- |
  * | 01 | call TvInit() -  Initialise and get a valid instance of the TV client | void | tvERROR_NONE | Should Pass |
- * | 02 | Call RegisterVideoContentChangeCB() - Register with the obtained data handle and a valid callback function | tvVideoContentCallbackData& | void | tvERROR_NONE | Should Pass |
+ * | 02 | Call RegisterVideoContentChangeCB() - Register with the obtained data handle and a valid callback function | tvVideoContentCallbackData * | void | tvERROR_NONE | Should Pass |
```

Comment:
Remove void test data, which is not required

Reply:
Done